src/main/java/com/mongodb/jdbc/utils/BsonUtils.java

```diff
@@ -35,6 +33,9 @@ public class BsonUtils {
     public static final JsonWriterSettings JSON_WRITER_SETTINGS =
             JsonWriterSettings.builder().outputMode(JsonMode.RELAXED).indent(true).build();
 
+    public static final JsonWriterSettings JSON_WRITER_NO_INDENT_SETTINGS =
```

Comment:
For logging purpose, having the info all on one line makes investigating the logs easier.